src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav></nav>
+    <div class="to-do-container">
+      <div class="header">
+        <p>Today's To Do</p>
+        <i class="fa-solid fa-arrows-rotate"></i>
+      </div>
+      <div id="updated-todo-container">
+        <div class="tdo-item">
+          <input
+            id="add-to-List"
+            type="text"
+            placeholder="Add to your list..."
+          />
+        </div>
+        <div id="to-do-list"></div>
+      </div>
+
+      <div class="last-row"><p>Clear all completed</p></div>
+    </div>
```

Comment:
I think it would be better to use semantic tags

Reply:
Thank you very much @Yorsyboy I will add the semantic tags. :hugs: